src/controlSurface.cpp

```diff
@@ -214,17 +212,18 @@ class Surface: public IReaperControlSurface {
 		postGoToTrack(0, track);
 	}
 
-	virtual int Extended(int call, void* parm1, void* parm2, void* parm3) override {
+	virtual int Extended(int call, void* parm1, void* parm2, void* parm3)
+		override {
```

Comment:
Oof. This is a bit ugly. Shouldn't the override be indented to continuation width at least so that the function body and the header are at different levels? It's not a huge deal, but it's not great.

Reply:
Slightly nasty indeed. We probably need an additional rule for this to avoid breaking.

Reply:
I actually reverted the continuation width as part of switching back to BlockIndent, but that's probably not what we want. It's back at 4 now.